pose_prediction/plugin.cpp

```diff
@@ -10,31 +12,62 @@ class pose_prediction_impl : public pose_prediction {
     pose_prediction_impl(const phonebook* const pb)
 		: sb{pb->lookup_impl<switchboard>()}
 		, _m_slow_pose{sb->subscribe_latest<pose_type>("slow_pose")}
+        , _m_imu_biases{sb->subscribe_latest<imu_biases_type>("imu_biases")}
         , _m_true_pose{sb->subscribe_latest<pose_type>("true_pose")}
     { }
 
-    virtual pose_type get_fast_pose() const override {
-		const pose_type* pose_ptr = _m_slow_pose->get_latest_ro();
-		return correct_pose(
-			pose_ptr ? *pose_ptr : pose_type{}
-		);
+    // No paramter pose predict will just get the current slow pose based on the next vsync
+    virtual fast_pose_type get_fast_pose() override {
+		return get_fast_pose(std::chrono::high_resolution_clock::now());
```

Comment:
Isn't the no parameter version supposed to return the pose for the next vsync? If we are not doing that because the timewarp vsync PR isn't here yet, then update the comment for now so it says that we return the pose for `now` (which may still be in the future with regards to the last IMU sample).